bridge_core.py

```
@@ -745,10 +761,50 @@ def safe_label(self, light):
+
+        Prefer the extended multizone API — its response length is the real
+        zones_count. The classic get_color_zones() over-reports on some firmware
+        (it returns the padded requested range), which showed up as an inflated
+        zone count in the UI. Falls back to classic when extended isn't supported.
+        """
+        try:
+            ext = light.extended_get_color_zones()
+            if ext:
+                return len(ext)
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)